app/lib/forms/base.rb

```diff
@@ -82,5 +82,16 @@ def reset_store!
     def query_store
       wizard.query_store
     end
+
+    def build_option_struct(value:, link_errors: false, divider: false, include_if: true, revealed_question: nil)
+      return unless include_if
```

Comment:
The param name `include_if` threw me a bit here. Until I looked at the code I wasn't sure what was being included.

Does it need to be passed in? Would conditionally adding it to the array work instead?

```
build_option_struct(value: "school", link_errors: true),
build_option_struct(value: "trust") if works_in_school? && inside_catchment?,
build_option_struct(value: "self"),
```

Or perhaps it could benefit from a name change (`render`?)